cmd/deletetree/main.go

```diff
@@ -37,7 +38,12 @@ func main() {
 	flag.Parse()
 	defer glog.Flush()
 
-	conn, err := grpc.Dial(*adminServerAddr, grpc.WithInsecure())
+	dialOpts, err := rpcflags.NewClientDialOptionsFromFlags()
+	if err != nil {
+		glog.Exitf("failed to determine dial options: %v", err)
```

Comment:
supernit: /failed/Failed/ (because it's a log line rather than an error)

Reply:
Done, also changed it on the rest of the file to keep it consistent.